modules/nf-core/bedtools/complement/main.nf

```diff
@@ -21,6 +21,7 @@ process BEDTOOLS_COMPLEMENT {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    if ("$bed" == "${prefix}.bed") error "Input and output names are the same, use \"task.ext.prefix\" to disambiguate!"
```

Comment:
Yes although this is a standard feature across all nf-core modules with the risk of a name collision. This would best be raised as an issue on `nf-core/modules` as we just update from there.